sensor_msgs/Image image
int8 colour
int8 shape
geometry_msgs/TransformStamped object_location
